content/docs/tasks/security/authz-tcp/index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Authorization for TCP services
+description: Shows how to set up role-based access control for TCP services.
+weight: 40
+keywords: [security,access-control,rbac,tcp,authorization]
+---
+
+This task covers the activities you might need to perform to set up Istio authorization, also known
+as Istio Role Based Access Control (RBAC), for TCP services in an Istio mesh. You can read more in
+[authorization](/docs/concepts/security/#authorization) and get started with a basic tutorial in
+Istio Security Basics.
+
+## Before you begin
+
+The activities in this task assume that you:
+
+* Understand [authorization](/docs/concepts/security/#authorization) concepts.
+
+* Have set up Istio on Kubernetes **with authentication enabled** by following the instructions in the
+  [quick start](/docs/setup/kubernetes/quick-start/), this tutorial requires mutual TLS to work. Mutual TLS
+  authentication should be enabled in the [installation steps](/docs/setup/kubernetes/quick-start/#installation-steps).
+
+## Setup Bookinfo with MongoDB TCP service
+
+The normal Bookinfo sample includes only HTTP services by default, in order to demonstrate Istio
+authorization for TCP services, we'll update the Bookinfo sample to use the `v2` of the `ratings`
+service which talks to a MongoDB backend.
+
+> If you are using a namespace other than `default`, use `kubectl -n namespace ...` to specify the namespace.
+
+1. Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application
+
+    After it has been deployed, point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). You should see:
+
+        * The "Book Details" section in the lower left part of the page, including type, pages, publisher, etc.
+        * The "Book Reviews" section in the lower right part of the page.
+
+    If you refresh the page several times, you should see different versions of reviews shown in the
+    product page, presented in a round robin style (red stars, black stars, no stars)
+
+1. Install `v2` of the `ratings` service with service account `bookinfo-ratings-v2`
+
+    In this task, we will enable access control using service accounts, which are cryptographically
+    authenticated in the mesh. In order to give different microservices different access privileges,
+    we will create the `v2` of the `ratings` with service account `bookinfo-ratings-v2`. Other services
+    will have a default service account `default`.
+
+    If you are using a cluster with automatic sidecar injection enabled,
+    simply deploy the service using `kubectl`:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/rbac/ratings-v2-add-serviceaccount.yaml@
```

Comment:
Do you need to create SA `bookinfo-rating-v2` before this step?

Reply:
This yaml already includes the SA `bookinfo-ratings-v2`